packages/sodium/lib/src/js/sodium_js_sumo_init.dart

```diff
@@ -0,0 +1,22 @@
+import 'dart:async';
+
+import '../api/advanced/advanced_sodium.dart';
+import 'api/advanced/advanced_sodium_js.dart';
+import 'bindings/sodium.js.dart';
+
+/// Static class to obtain a [Sodium] instance.
+///
+/// **Important:** This API is is different depending on whether it is used from
+/// the VM or in transpiled JavaScript Code. See the specific implementations
+/// for more details.
+abstract class SodiumInit {
```

Comment:
Update documentation for the whole file. Also, the simple "initSumo" is missing.

Do you think it would be useful to add a simply test, if the given JS object is actually a sumo instance? for example, by checking if a certain sumo API is available?